package.json

```diff
@@ -5,7 +5,9 @@
   "scripts": {
     "build": "ember build",
     "start": "ember server",
-    "test": "ember try:each"
+    "test": "npm run test-lib && npm run test-ember",
+    "test-ember": "ember try:each",
+    "test-lib": "tape tests-node/**/*-test.js"
```

Comment:
Don't have strong opinions on the directory structure and framework in use — this was just quick to get up and running. Might have been nice to put these in a subdirectory of `tests`, but I couldn't immediately come up with a way to filter those out from attempting to also run in the browser.